base_search_custom_field_filter/models/base.py

```diff
@@ -49,15 +49,22 @@ def get_view(self, view_id=None, view_type="form", **options):
     def get_views(self, views, options=None):
         """Inject fake field definition for having custom filters available."""
         res = super().get_views(views, options)
+
+        if self._name not in res["models"]:
+            res["models"][self._name] = {}
+
         custom_filters = self.env["ir.ui.custom.field.filter"].search(
             [("model_name", "=", self._name)]
         )
+
         for custom_filter in custom_filters:
             field = custom_filter._get_related_field()
-            field_name = custom_filter.expression
-            res["models"][self._name][field_name] = field.get_description(self.env)
-            # force this for avoiding to appear on the rest of the UI
-            res["models"][self._name][field_name]["selectable"] = False
-            res["models"][self._name][field_name]["sortable"] = False
-            res["models"][self._name][field_name]["store"] = False
+            if field:
```

Comment:
For reducing the diff, do the negative one `if not field: continue`. Anyway, why do you need this safeguard? Please document it through a code comment.